contracts/oracle.tact

```diff
@@ -43,13 +47,13 @@ message Refund {
 }
 message Tick {
     expireAt: Int as uint256; // If now() is larger than expireAt, then return the funds back to the Watchmaker if value is enough to pay the gas
-    baseAssetPrice: Int as uint256; // baseAsset price relative to quoteAsset
+    baseAssetPrice: Int as uint256; // baseAsset price relative to quoteAsset in float type
 }
 message Tock {
     alarmIndex: Int as uint256; // the interacted alarm index
-    scale: Int as uint32;
-    createdAt: Int;
-    watchmaker: Address;
+    scale: Int as uint32; // The alarm's scale, it always equals to 1 when creating a new alarm
```

Comment:
The scale of the "new alarm" is not necessarily 1; there is an issue with the annotation here. For instance, alarms created after wind events will have a scale greater than 1.